templates/CRM/Admin/Form/Preferences/Mailing.hlp

```diff
@@ -14,3 +14,11 @@
   <p>{ts}Specify an Email From Address to use when the system sends an email but a reply is not expected, for example when a user is sent an email for a double opt-in.{/ts}</p>
   <p>{ts}Leaving this blank will use the default which will be do-not-reply@default_domain where the default_domain will be the email domain address of your default mailing account also used for bounce handling.{/ts}</p>
 {/htxt}
+{htxt id="smtp_450_is_permanent-title"}
+  {ts}Treat SMTP Error 450 4.1.2 as permanent.{/ts}
+{/htxt}
+{htxt id="smtp_450_is_permanent"}
+  <p>{ts}Some SMTP servers are configured to not accept an email if the recipient cannot be reached. Most of the time it means that the domain doesn't exist at all and the SMTP server returns a 5xx permanent error.{/ts}</p>
+  <p>{ts}There're other cases when the domain name is actually registered but its name servers are not reachable, or they don't give a meaningful answer. In these cases the SMTP server returns a 450 temporary delivery error and this is when CiviCRM gets stuck trying a trying to deliver those messages.{/ts}</p>
+  <p>{ts}This option it is useful to decide how to handle these cases, whether to follow the "standard" or consider them as permanent failures.{/ts}</p>
```

Comment:
Since these are separated into their own `ts`'s it loses the context for the translator - it's not obvious what "this option" means without the rest of the text. I would just do this as one whole paragraph. Also just a typo or two. How about:

```
<p>{ts}Some SMTP servers are configured to not accept an email if the recipient cannot be reached. Most of the time it means that the domain doesn't exist at all and the SMTP server returns a 5xx permanent error. But sometimes the domain name is actually registered but its name servers are not reachable, or they don't give a meaningful answer. In these cases the SMTP server returns a 450 temporary delivery error and the mailing job gets stuck in status Running. Enabling this option will consider them as permanent failures.{/ts}</p>
```

Otherwise, PR looks good. It's a little hard to test but runs as expected - I had to fake a pear_error.

Reply:
thanks @demeritcowboy for your review.

About how to test I'd have a couple of e-mails that give this message, but I dont't think it's appropriate to write them here. What do you think?

Reply:
I think we're good. Thanks.